ui/cryptomaterial/image.go

```diff
@@ -57,7 +57,7 @@ func (img *Image) LayoutSize(gtx C, size unit.Dp) D {
 }
 
 func (img *Image) LayoutSize2(gtx C, width, height unit.Dp) D {
-	dst := image.NewRGBA(image.Rectangle{Max: image.Point{X: int(width * 2), Y: int(height * 2)}})
+	dst := image.NewRGBA(image.Rectangle{Max: image.Point{X: int(width), Y: int(height * 2)}})
```

Comment:
This reintroduces this issue

![Screenshot from 2023-09-29 10-44-42](https://github.com/crypto-power/cryptopower/assets/25265396/8dbe3a19-d5b6-4639-a175-8d9c9ee1ab1c)